scheduler/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.18.0] - 2018-06-25
+### Added
+- Container fields to /jobs, from @dposada
+- reason_mea_culpa to instance responses, from @dposada
+- Support for x-forwarded-proto header for CORS requests, from @pschorf
+
```

Comment:
Should we also mention:

- removed rebalancer's cluster utilization check (and corresponding `:rebalancer` `:min-utilization-threshold` config)
- removed `:mesos` `:master-hosts` config